src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,46 @@
+.stars {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 16px 0;
+
+  &__container {
+    display: flex;
+  }
+
+  &__star::before {
+    content: '';
+    display: flex;
```

Comment:
Using 'display: flex;' on a pseudo-element (::before) is unusual and might not have the intended effect. Pseudo-elements are typically used for styling content, not layout.